content/migrate/tf2pulumi.md

```diff
@@ -0,0 +1,633 @@
+        case "url":
+            post.data = JSON.stringify({ url: tfUrl, language: language });
+            break;
+        default: // "code"
+            post.data = JSON.stringify({ code: tfCode, language: language });
+            break;
+        }
+        post.dataType = "json";
+    }
+    $.post(post)
+        .done(function(data) {
+            clearLanguageFiles(languageTextbox);
+
+            if (data.files) {
+                let filenames = Object.keys(data.files);
+                filenames.sort();
+                for (let i = 0; i < filenames.length; i++) {
+                    let fn = filenames[i];
+                    let code = data.files[fn];
+                    addLanguageFile(languageTextbox, fn, code);
+                }
+                $("#pulumi-code-download-icon").show();
+                $("#pulumi-code-download-button").removeClass([ "opacity-50", "cursor-not-allowed" ]);
+            } else {
+                displayCouldNotConvert(languageTextbox);
+            }
+
+            if (data.diagnostics) {
+                $("#pulumi-warnings").text(data.diagnostics);
+                $("#pulumi-warnings").show();
+            }
+        })
+        .fail(function(err) {
+            let errorText = "An unspecified error occurred";
+            if (err) {
+                if (err.responseText) {
+                    errorText = err.responseText;
+                    try {
+                        let errdata = JSON.parse(err.responseText);
+                        if (errdata.error) {
+                            errorText = errdata.error;
+                        }
+                    } catch {
+                        // ignore.
+                    }
+                }
+                errorText += " [" + err.statusText + " " + err.status + "]";
+            }
+            $("#pulumi-errors").text(errorText);
+            $("#pulumi-errors").show();
+            displayCouldNotConvert(languageTextbox);
+        }).
+        always(function() {
+            $(document.body).css({ "cursor": "default" });
+            $("#terraform-code, #terraform-url").css({ "cursor": "text" });
+            $("pulumi-chooser[type='language'] > ul > li > a").css({ "cursor": "pointer" });
+        });
+}
+
+// downloadCode downloads the currently converted code, if available.
+function downloadCode() {
+    let zip = new JSZip();
+    for (let fn of Object.keys(currentCodeFiles)) {
+        zip.file(fn, currentCodeFiles[fn]);
+    }
+    zip.generateAsync({ type: "blob" }).then(function(content) {
+        // Use FileSaver.js to save the file, triggering download in the user's browser.
+        saveAs(content, "tf2pulumi.zip");
+    });
+}
+
+function getCannedExample(id) {
+    if (id === undefined) {
+        // Look up the currently selected example ID.
+        id = $("#terraform-canned-example").children("option:selected").attr("id");
+    }
+
+    let comment = "#"; // to suppress Markdown lint errors.
+    switch (id) {
```

Comment:
Sounds like this switch was causing some sentry alerts earlier. We should pick up the fix from here https://github.com/pulumi/docs/pull/4012

Reply:
Great, thanks for catching this -- fixed.